components/LoginForm/index.tsx

```diff
@@ -36,7 +36,9 @@ const LoginForm: React.FC<LoginFormProps> = (props) => {
       />
       <Button
         title="Log In"
-        onPress={onLogin}
+        onPress={() => {
+          if (onLogin) onLogin(email, password);
+        }}
```

Comment:
xd. { }

Reply:
Unlike the previous two requests, I believe braces are not needed here because the line is too short, and the if statement is only there as a sanity check for making sure that the `onLogin` prop is not undefined; all the logic should happen inside `onLogin`, and there should be no other logic that depend on the defined condition of `onLogin`.